src/components/tabs/js/tabDirective.js

```diff
@@ -21,6 +21,7 @@
  *
  * @param {string=} label Optional attribute to specify a simple string as the tab label
  * @param {boolean=} ng-disabled If present and expression evaluates to truthy, disabled tab selection.
+ * @param {string=} md-class-name Optional attribute to specify a class name for the tab button
```

Comment:
For consistency with other APIs like `md-menu-class`, `md-menu-container-class`, and `md-input-class`, we should probably call this one `md-tab-class`.